lib/dialects/postgres/connection-manager.js

```diff
@@ -129,7 +130,12 @@ ConnectionManager.prototype.connect = function(config) {
     }
 
     if (!self.sequelize.config.keepDefaultTimezone) {
-      query += 'SET client_min_messages TO warning; SET TIME ZONE INTERVAL \'' + self.sequelize.options.timezone + '\' HOUR TO MINUTE;';
+      var isZone = !!moment.tz.zone(self.sequelize.options.timezone);
```

Comment:
Yes it does. It returns an `Object` when there is a valid zone otherwise `null`.